pyproject.toml

```diff
@@ -1,2 +1,4 @@
 [build-system]
 requires = ["setuptools", "wheel", "numpy", "cython", "mpi4py"]
+[tool.black]
```

Comment:
Are we blacking again @Lun4m? 😂

Reply:
Nooo, on the contrary that's supposed to prevent any `black` formatting in the repo-wide 😅